targets/psa_switch/psa_switch.cpp

```diff
@@ -563,6 +563,39 @@ PsaSwitch::egress_thread(size_t worker_id) {
     Deparser *deparser = this->get_deparser("egress_deparser");
     deparser->deparse(packet.get());
 
+    // egress cloning - each cloned packet is a copy of the packet as output by the egress deparser
+    auto clone = phv->get_field("psa_egress_output_metadata.clone").get_uint();
+    if (clone) {
+      MirroringSessionConfig config;
+      auto clone_session_id = phv->get_field("psa_egress_output_metadata.clone_session_id").get_uint();
+      auto is_session_configured = mirroring_get_session(static_cast<mirror_id_t>(clone_session_id), &config);
```

Comment:
should be:
```cpp
auto clone_session_id = phv->get_field("psa_egress_output_metadata.clone_session_id").get< mirror_id_t>();
auto is_session_configured = mirroring_get_session(clone_session_id, &config);
```

please "fix" it for ingress cloning as well

Reply:
done. adding in next commit